vipps-login-api-quick-start.md

```diff
@@ -7,62 +7,59 @@ END_METADATA -->
   * [Step 3: Set up Postman environment](#step-3-set-up-postman-environment)
 * [Make API calls](#make-api-calls)
-
 <!-- END_TOC -->
 
-Document version 1.0.3.
+Document version 1.0.4.
 
 ## Postman
 
 ### Prerequisites
 
-* If you are not familiar with Postman, review
-[Getting started with Postman](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/vipps-quick-start-guides#getting-started-with-postman).
-
-* You will need a merchant test account.
-If you haven't already set this up, see
-[Test Merchants](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/vipps-test-environment#test-merchants).
+Review
+[Vipps quick start guides](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/vipps-quick-start-guides) for information about getting your test environment set up.
 
-### Step 1: Get the Postman collection
+### Step 1: Get the Vipps Postman collection and environment
 
-Import the collection by following the steps below:
+Save the following files to your computer:
 
-1. Click `Import` in the upper-left corner.
-2. Import the [vipps-login-api-postman-collection.json](tools/vipps-login-api.postman_collection.json) file.
+* [Vipps Login API Postman collection](tools/vipps-login-api.postman_collection.json)
+* [Vipps Login API Postman environment](tools/vipps-login-api.postman_environment.json)
 
-### Step 2: Import the Postman environment
+### Step 2: Import the Vipps Postman files
 
-1. Click `Import` in the upper-left corner.
-2. Import the [vipps-login-api.postman_environment.json](tools/vipps-login-api.postman_environment.json) file.
+1. In Postman, click *Import* in the upper-left corner.
+1. In the dialog that opens, with *File* selected, click *Upload Files*.
+1. Select the two files you have just downloaded and click *Import*.
 
 ### Step 3: Set up Postman environment
 
 1. Click the down arrow, next to the "eye" icon in the top-right corner, and select the environment you have imported.
 1. Click the "eye" icon and, in the dropdown window, click `Edit` in the top-right corner.
 1. Fill in the `Current Value` for the following fields to get started.
-   - `well-known_uri`
-   - `client_id`
-   - `client_secret`
-   - `redirect_uri`
+   * `client_id` - Merchant key is required for getting the access token.
+   * `client_secret` - Merchant key is required for getting the access token.
+   * `well-known_uri` - URL to well-known endpoint for used environment. In the test environment, you can use `https://apitest.vipps.no/access-management-1.0/access/.well-known/openid-configuration`. See [API Guide: well known](vipps-login-api#openid-connect-discovery-endpoint) for more details.
+   * `redirect_uri` - Redirect URL which the user agent is redirected to after finishing a login. The URL must be exactly the same as the one specified on portal.vipps.no. This URI needs to be pre-registered with Vipps and supplied as a query parameter on calls to the OAuth2 authorize endpoint.
```

Comment:
Is there an example we can give? I also don't know what I can use here, to make it work.